### DISCUSSION TITLE
Why should I want/advise to use Jamulus with Jack on Windows 10/11?

### DISCUSSION THREAD

**Discussion by bugith** [jamulussoftware/jamulus#3202] [General]
Hi
Please give lights/scenarii pro/cons or links in the doc that explain why one want to use the version of Jamulus Windows 10/11 (maybe elder?) that is suffixed **_jack.exe**.
Does the [setup binary](https://github.com/jamulussoftware/jamulus/releases/download/r3_10_0/jamulus_3.10.0_win_jack.exe) also install Jack-on-Windows so we don't need to [install](https://jackaudio.org/downloads/) it as a separate step?
If no, then will the Jackified version work without Jack for Win in the case we want to try/use Jack later?
I understand this is not really the place to talk about the difference between all Windows sound servers available, but some examples or discussion here may be a kind way to learn general basic sound/OS concepts environment... by doing... around Jamulus.
I ask this because I'm a linux user and I'd like to give advice to friends (Although they use Windows, they are friends ;-) ). I now I could try with a VM to answer to myself but I'm afraid I'll fall in traps with USB sharing or at least with performances issues.

Happy birthday Volker Fisher and many thanks to the few main and many follower devs for this great piece of soft and the huge effort on web site and documentation since the mid 2000's.

**Comment by ann0see** [comment]
Hi,

Thanks for asking. The JACK version on Windows is mainly for people who already know how to use JACK/Prefer a fully FLOSS version.
JACK has some advantages like better routing capabilities (in my opinion) but also feels less plug and play, as you need to start the JACK server before starting Jamulus. 
As with ASIO, we don't install JACK automatically, but warn and link the user to the correct pages if JACK is not found.

The Server works without JACK/ASIO, but there's no support for the client without any sound service.
@henkdegroot or @jujudusud might have some valuable insights on this topic too.

**Reply by bugith** [reply]
Hi Ann0see

I gave a try with a VM with USB2 controler plus shared my UMC404HD and installed the manufacturer driver (ASIO compatible) plus jack for win (first w/o and then with Jack-router ASIO driver) and the jackified Jamulus but jackd never started from qjackctl :-( throwing errors like server already running (even though I killed all instances, or Cannot connect to named pipe after wait = \\.\pipe\server_jack_default_0 err = 2). Well I guess it's definitively not the place for this here and VM adds unwanted complexity when it comes to explore how_to this and that.
Oh yes, I thought about FLOSS but forgot to speak about it. Is there a Windows native or other third party thing like qjackctl and its patchbay?

**Reply by ann0see** [reply]
Yes, there is QJackCtl. I think you can install it while installing JACK. Maybe it's even already installed.

**Reply by bugith** [reply]
Yes ann0see, this is Jack for win setup/install specific : QJackCtl is a pre-selected option, while Jack-router ASIO driver option is not.
My question was if alternative to QJackCtl exists or something inside Windows that do the same job as Jack+QJackCtl

**Reply by ann0see** [reply]
I don't think windows supports managing jack natively, but you should probably ask on the JACK repository. 

With ASIO there ARE different routing programs available. Reaper as DAW allows routing (Should be named Rearoute or similar). 
Personally, I'd prefer JACK because it looks like a physical board. 

@pljones might know someone from WorldJam. 

Otherwise have a look at https://jamulus.io/kb/index.html
and https://jamulus.io/wiki/Tips-Tricks-More#learning-about-remote-band-rehearsing

**Comment by dcorson-ticino-com** [comment]
If you are just running a client with a sound card that has ASIO support I see no reason to run Jack.
It may be helpful if you need specific routing with other applications, but you do need to make yourself a checklist how to get everything up and running in the right order, etc.

**Reply by bugith** [reply]
Thanks @dcorson-ticino-com well I think this is the answer I hoped : only Jamulus with ASIO support => No need of Jack.
If I need any other windows audio application running at the same time then I'll need Jack, right ?

BTW, would using a USB remote modeling control for a Yamaha thr30-ii amp (48kHz ready and ASIO compatible) is considered as an application that would require jack routing with Jamulus?
Thanks for interest

**Reply by dcorson-ticino-com** [reply]
I don't know either what a remote modeling control is nor a Yamaha xxyy amp, but if the remote modeling control is just a remote to control something in the amp it would not be in the audio path and not require routing.  
But it depends on the details probably.

**Reply by bugith** [reply]
I wandered because both audio and modeling controls use the same USB port

**Reply by ann0see** [reply]
Not sure if I understand you correctly,
But:
1. Jamulus UI can be controlled by a mixer via MIDI
2. Jamulus can be connected via ASIO and JACK to applications which do routing in software
3. You can connect analog gear which has two inputs and two outputs (including hardware mixers) which support ASIO.

**Comment by bugith** [comment]
Thank you all
